unixfs/io/dirbuilder.go

```diff
@@ -157,6 +162,9 @@ func (d *Directory) Links(ctx context.Context) ([]*ipld.Link, error) {
 	return d.shard.EnumLinks(ctx)
 }
 
+// Find returns the ipld.Node with the given name, if it is contained in this
+// directory. Find only searches in the most inmediate links, and not
+// recursively in the tree.
 func (d *Directory) Find(ctx context.Context, name string) (ipld.Node, error) {
```

Comment:
Find returns the root node of the file named `name` within this directory (really, it should be called `Lookup`). It does, in fact, traverse the tree (when searching a hamt).